microraft-hocon/build.gradle.kts

```
@@ -0,0 +1,85 @@
+plugins {
+    `java-library`
+    alias(libs.plugins.defaults)
+    alias(libs.plugins.metadata)
+    alias(libs.plugins.javadocLinks)
+}
+
+group = "io.microraft"
+version = "0.9-SNAPSHOT"
+description = ""
+
+metadata {
+    moduleName.set("io.microraft.microraft-hocon")
```

```suggestion
    moduleName = "io.microraft.microraft-hocon"
```
please use simple assignment in all build scripts